cypress/support/graphql-test-utils.js

```diff
@@ -2,6 +2,7 @@
 export const hasOperationName = (req, operationName) => {
   const { body } = req;
   return (
+    // eslint-disable-next-line no-prototype-builtins
```

Comment:
Can you clarify why this needed to be added? Is it just something that makes GoLand complain?

Reply:
This is per Clay's suggestion to suppress an unrelated linting error